Bottom: aebfe687f38fa1db26dd8a491501ca9f2e89abf8
Top:    e7432125e2f8178d24cad3a675f55ffd94e50e24
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-18 18:40:47 +0000

polygon.h: Reduce circ_segs to 8 for debug clarity


---

diff --git a/src/polygon.h b/src/polygon.h
index 369efea..0643765 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 40
+#define POLY_CIRC_SEGS 8 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
